src/modules/template-details/TemplateDetails.tsx

```diff
@@ -12,7 +12,8 @@ import { Button, Loader, More, Tags } from 'ui';
 
 import { convertDate } from 'utils';
 
-import { Template } from 'core/api';
+import { Template, forkTemplate } from 'core/api';
+import { ForkTemplateInfoContext } from 'shared/providers/fork-template-info'
```

Comment:
This should be useTemplateInfoProvider hook instead of Context

Reply:
done